src/Octoshift/AdoApi.cs

```diff
@@ -13,8 +13,13 @@ namespace OctoshiftCLI
     public class AdoApi
     {
         private readonly AdoClient _client;
+        private readonly string _adoBaseUrl;
 
-        public AdoApi(AdoClient client) => _client = client;
+        public AdoApi(AdoClient client, string adoServerUrl)
+        {
+            _client = client;
+            _adoBaseUrl = adoServerUrl?.Trim('/');
```

Comment:
nit
```suggestion
            _adoBaseUrl = adoServerUrl?.TrimEnd('/');
```

Reply:
not sure if it's a good idea but wouldn't it make more sense to name this `adoServerBaseUrl`?

Reply:
it's also the base URL when hitting ado SERVICE (i.e. https://dev.azure.com)